external/glfw/lib/ios/app/AppDelegate.m

```diff
@@ -84,6 +84,18 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
     window = [[UIWindow alloc] initWithFrame:bounds];
     window.rootViewController = [[[ViewController alloc] init] autorelease];
+
+    // Load the LaunchScreen storyboard
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"LaunchScreen" bundle:nil];
```

Comment:
Agreed, we need to look into the plist properties after the `launch-screen` property before accepting this PR.